src/test/java/jUnit/JsonParserTest.java

```diff
@@ -12,27 +11,35 @@
 import java.io.*;
 import java.nio.file.Files;
 import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-import static jUnit.Utils.Constants.FILE_NAME_LIST;
 import static org.junit.jupiter.api.Assertions.*;
 
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class JsonParserTest {
 
-    static Cart initialCart;
-    static JsonParser parser;
+    Cart initialCart;
+    JsonParser parser;
```

Comment:
as for the comment below - instantiate JsonParser here. 
Additionally, I would suggest using private access modifier